srcpkgs/onedrive/files/onedrive/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec onedrive --monitor
```

Comment:
Should this service be run as root?

Reply:
that's what maintaners seem to encourage here: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#running-onedrive-as-a-system-service, but I can change that

Reply:
If this is not intended to be run as root (and I suspect it isn't), the service should be removed. You can define a [per-user service](https://docs.voidlinux.org/config/services/user-services.html) to manage this locally, but there is no mechanism to support per-user services in XBPS packages.

Reply:
They have instructions for running as user, which is the bet way, IMO. https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#onedrive-service-running-as-a-non-root-user-via-systemd-with-notifications-enabled-arch-ubuntu-debian-opensuse-fedora

Running something like this as root isn't great practice. I don't think we should include the service.

Reply:
The "run as root" directions actually include `systemctl --user`, which is *not* a system-level service and will *not* run as root. It will run as the user that invokes `systemctl`. Down the page, there are instructions that load a system-level service as root, but that seems like a terrible idea to sync user-level files.

My suspicion is that the authors of this package don't thoroughly understand `systemd` task management.

Reply:
the service is removed now